examples/enum_in_args.rs

```diff
@@ -19,7 +19,11 @@ arg_enum! {
 #[derive(StructOpt, Debug)]
 struct Opt {
     /// Important argument.
-    #[structopt(raw(possible_values = "&Baz::variants()", case_insensitive = "true"))]
+    #[structopt(raw(
+        possible_values = "&Baz::variants()",
+        case_insensitive = "true",
+        default_value = "\"FooBar\"",
+    ))]
```

Comment:
It complexify the example.

IMHO, the correct way to do that would be (today, not supported by the minimal rust version)
```rust
 #[derive(StructOpt, Debug)] 
struct Opt {
    #[structopt(
        raw(possible_values = "&Baz::variants()"),
        case_insensitive = true,
        default_value = "FooBar"
    )]
    i: Baz,
}
```

Reply:
Yes, that's exactly the point. You _would want_ the simpler version to work, so you type in `default_value = "FooBar"`. This is the compilation error that you get now:

```
error[E0425]: cannot find value `FooBar` in this scope
  --> src/cli_args.rs:14:10
   |
14 | #[derive(StructOpt, Debug)]
   |          ^^^^^^^^^ not found in this scope

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
error: Could not compile `rua`.
```

What does it mean? How to fix it? The fix was not immediately obvious to me and only came to me later, accidentally. By having this documented I hope to help people fight with the potential problem. Be aware of the problem and the fix rather than hiding both.

Reply:
And the version I gave is working, you just need a recent rust version.

Reply:
@TeXitoi ah, indeed, I just missed the braces placement for `raw()`!

Do you know if it's possible to change the old form of
```
#[structopt(raw(possible_values = "&Baz::variants()", case_insensitive = "true"))]
```
to
```
#[structopt(raw(possible_values = "&Baz::variants()"), case_insensitive = true)]
```
in the old compiler then? This way, just appending `, default_value = "whatever"` at the end would not break anything.

Or does it feel more like an edge case, and many other developers clearly understand the difference between inside `raw` and outside of it?

Reply:
I think that's not possible with rust 1.21 which is the current oldest supported version by structopt. But this version might change within a few month.

Reply:
Thanks for all explanations!

Last question, shouldn't the import be `use structopt::clap::arg_enum` instead of `use clap::arg_enum`?

Reply:
I think it doesn't work on older version because of macro reexport limitation, but that's better to do so if possible.